multi/reporter.go

```diff
@@ -167,13 +208,46 @@ func (m multiMetric) ReportTimer(interval time.Duration) {
 	}
 }
 
+func (m multiMetric) ValueBucket(
+	bucketLowerBound, bucketUpperBound float64,
+) tally.CachedHistogramBucket {
+	var multi []tally.CachedHistogramBucket
+	for _, m := range m.histograms {
+		multi = append(multi,
+			m.ValueBucket(bucketLowerBound, bucketUpperBound))
+	}
+	return multiHistogramBucket{multi}
+}
+
+func (m multiMetric) DurationBucket(
+	bucketLowerBound, bucketUpperBound time.Duration,
+) tally.CachedHistogramBucket {
+	var multi []tally.CachedHistogramBucket
+	for _, m := range m.histograms {
+		multi = append(multi,
+			m.DurationBucket(bucketLowerBound, bucketUpperBound))
+	}
+	return multiHistogramBucket{multi}
+}
+
+type multiHistogramBucket struct {
+	multi []tally.CachedHistogramBucket
+}
+
+func (m multiHistogramBucket) ReportSamples(value int64) {
+	for _, m := range m.multi {
+		m.ReportSamples(value)
+	}
+}
+
 type multiBaseReporters []tally.BaseStatsReporter
 
 func (r multiBaseReporters) Capabilities() tally.Capabilities {
 	c := &capabilities{reporting: true, tagging: true}
 	for _, r := range r {
 		c.reporting = c.reporting && r.Capabilities().Reporting()
 		c.tagging = c.tagging && r.Capabilities().Tagging()
+		c.histograms = c.tagging && r.Capabilities().Histograms()
```

Comment:
Should this be `c.histograms`, and should the seed value set `histograms` to true?

Does this have any test coverage?